python/cudf_polars/cudf_polars/dsl/expressions/rolling.py

```diff
@@ -62,6 +78,15 @@ def to_request(
     return plc.rolling.RollingRequest(col.obj, min_periods, value.agg_request)
 
 
+def _by_exprs(b: Expr | tuple) -> list[Expr]:
+    if isinstance(b, Expr):
+        return [b]
+    if isinstance(b, tuple):  # pragma: no cover; tests cover this path when
+        # run with the distributed scheduler only
+        return [e for item in b for e in _by_exprs(item)]
+    return [expr.Literal(DataType(pl.Int64()), b)]  # pragma: no cover
```

Comment:
Note: Covered by the streaming tests when run with the distributed scheduler, but we don't enforce code coverage for that run, so we skip it.